.gitignore

```diff
@@ -1,7 +1,10 @@
 _site
 .sass-cache
 .jekyll-metadata
-assets/uswds
+assets/img
+assets/css
+assets/fonts
+assets/js
```

Comment:
Adding assets in a directory that mirrors how code outputs to uswds-site to facilitate copy/paste